custom/data.ts

```diff
@@ -1,73 +1,72 @@
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
+  text: `Bringing exploratory mindset to technical leadership,
+        with six years of experience in software development and computational research`,
 };
 
 const projects = [
   {
-    title: faker.lorem.words(2),
-    text: faker.lorem.words(5),
+    title: "Software development",
+    text: "Full stack development description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
+        title: "Portfolio",
+        text: "A portfolio project to show my skills and existing projects",
+        link: "https://github.com/raiaqt/betterai-portfolio",
         year: "2024",
         github: true,
         image: "placeholder.png",
       },
     ],
   },
   {
-    title: faker.lorem.words(3),
-    text: faker.lorem.words(4),
+    title: "Competitive programming",
+    text: "Competitive programming description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "Kattis submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/kattis-submissions",
         github: true,
         image: "placeholder.png",
       },
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "OnlineJudge submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/onlinejudge-submissions",
         github: true,
       },
     ],
   },
   {
-    title: faker.lorem.words(3),
-    text: faker.lorem.words(4),
+    title: "Writing",
+    text: "Writing description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
-        github: true,
+        title: "Milton Today",
+        text: "Milton Brewery Workers Organize a Strike to Protest Against Management",
+        link: "https://miltontoday.com.au/milton-brewery-workers-organize-strike-protest-management/",
+        year: "2019",
+        github: false,
       },
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
-        github: true,
+        title: "Indooroopilly News",
+        text: "Morningside’s Cate Campbell Aims for Gold in Gold Coast 2018",
+        link: "https://morningsidenews.com.au/morningsides-cate-campbell-aims-gold-gold-coast-2018/",
+        year: "2019",
+        github: false,
       },
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
```

Comment:
Consider updating the links with your personal information for consistency and correctness.

Suggested change:
```javascript
 instagram: "https://www.instagram.com/raiaqt/",
 github: "https://github.com/raiaqt",
 linkedin: "https://www.linkedin.com/in/raia-quitoriano/",
 gmail: "mailto:rsquitoriano@gmail.com",
``` 

For reference, see [Creating Links in HTML](https://www.w3schools.com/html/html_links.asp).